a10_octavia/controller/worker/controller_worker.py

```diff
@@ -322,10 +322,21 @@ def delete_load_balancer(self, load_balancer_id, cascade=False):
         if vthunder:
             deleteCompute = self._vthunder_repo.get_delete_compute_flag(db_apis.get_session(),
                                                                         vthunder.compute_id)
-        if cascade:
-            (flow, store) = self._lb_flows.get_delete_load_balancer_flow(lb, deleteCompute, True)
+
+        if lb.project_id in CONF.hardware_thunder.devices:
+            if cascade:
```

Comment:
here cascade code is duplicated, we can take out cascade code and set cascade flag and pass that flag to respective flows.

Reply:
Done